web/src/enterprise/campaigns/global/marketing/CampaignsSiteAdminMarketingPage.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import { CampaignsMarketing } from './CampaignsMarketing'
+import { Link } from '../../../../../../shared/src/components/Link'
+
+export interface CampaignsSiteAdminMarketingPageProps {}
+
+export const CampaignsSiteAdminMarketingPage: React.FunctionComponent<CampaignsSiteAdminMarketingPageProps> = () => (
+    <CampaignsMarketing
+        body={
+            <section className="my-3">
+                <h2>Get started</h2>
+
+                <p>Creating and managing campaigns is only available to Sourcegraph admins at this time.</p>
+                <ol>
+                    <li>
+                        <a href="https://docs.sourcegraph.com/user/campaigns">
+                            Update your site configuration settings
+                        </a>{' '}
+                        to enable Campaigns for site admins.
+                        <div className="alert alert-info mt-3">
+                            <p>
+                                <strong>
+                                    By enabling this feature you agree to test out functionality that is currently in
+                                    beta, and acknowledge the following:
+                                </strong>
+                            </p>
+                            <ul className="mb-3">
+                                <li>
+                                    During the beta period Campaigns are free to use. After the beta period ends,
+                                    Campaigns will be a paid add-on.
+                                </li>
+                                <li>
+                                    Campaigns are not included as part of the Enterprise or Enterprise plus license
+                                    tiers.
+                                </li>
+                            </ul>
+                            Please <a href="mailto:sales@sourcegraph.com">contact us</a> for more information.
+                        </div>
+                    </li>
+                    <li>
+                        Optional: enable read-only access for all users.
+                        <div className="alert alert-warning mt-3">
+                            <strong>WARNING:</strong> Repository permissions are NOT enforced if you enable read-only
+                            views on campaigns. Any logged in user will be able to see all code changes associated with
+                            a campaign. Therefore, read-only access is recommended for instances without repository
+                            permissions configured.
+                        </div>
+                    </li>
+                    <li>
+                        <a href="https://docs.sourcegraph.com/user/campaigns#creating-campaigns">
```

Comment:
```suggestion
                        <a href="https://docs.sourcegraph.com/user/campaigns">
```

The new docs will have a "Are you a new Campaigns user section?" so I think we should link them to the docs entry point.